### DISCUSSION TITLE
Input files

### DISCUSSION THREAD

**Discussion by esraabenesm** [GreenleafLab/ArchR#1978] [Questions / Documentation]
Hello everyone, 

I have a small problem and I hope someone will have the answer. 

For my snATAC-seq analysis, I would like to load 2 runs per patient (so 2 input files for each patient), but I don't know how to make ArchR understand that the data in the two files correspond to one and the same sample.
knowing that I'm using the names() function to name the input files.
I'm asking because my results seem inconsistent afterwards. 

Maybe someone will have the solution. Thank you for taking the time to answer my question.

**Comment by rcorces** [comment]
A single Arrow file is created from each single-cell reaction (for ex. well of the 10x chip). There is no way around this. However, after ArchRProject creation, it is perfectly reasonable to create a new column in cellColData to annotate which sample a reaction comes from.
